helix-rest/src/test/java/org/apache/helix/rest/server/TestResourceAssignmentOptimizerAccessor.java

```diff
@@ -243,6 +246,8 @@ public void testComputePartitionAssignmentWaged() throws IOException {
         });
     Set<String> hostSet = new HashSet<>();
     resourceAssignments.forEach((k, v) -> v.forEach((kk, vv) -> hostSet.addAll(vv.keySet())));
+    // Assert the
```

Comment:
The comment is not complete here.

Reply:
TFTR. 
Updated. It was a careless mistake...